lib/glucose-stats.js

```diff
@@ -0,0 +1,244 @@
+
+
+const moment = require('moment');
+
+const log = console.error;
+
+/* eslint-disable-next-line no-unused-vars */
+const error = console.error;
+const debug = console.error;
+
+module.exports = {};
+const calcStatsExports = module.exports;
+
+// Calculate the sum of the distance of all points (overallDistance)
```

Comment:
Should this be sod instead of overallDistance?

Reply:
yes. Thanks!